test/test.js

```diff
@@ -218,14 +256,14 @@ const register = () => {
   server.route({
     method: 'GET',
     path: '/query-params',
-    config: {
+    options: {
       validate: {
         query: {
-          testDate: Joi.date(),
-          testArray: Joi.array(),
-          testObject: Joi.object(),
-          page: Joi.number(),
-          limit: Joi.number()
+          testDate: testJoi.date(),
+          testArray: testJoi.array(),
+          testObject: testJoi.object(),
+          page: testJoi.number(),
+          limit: testJoi.number()
```

Comment:
This is just a change that's required in test, right?

Reply:
https://github.com/hapijs/joi/issues/2031

⬆️ Due to this change, by default, Joi will not automatically coerce strings containing arrays or objects into those data-types. The update to the test reflects that any package using `hapi-pagination` will have to accommodate this change (which would have been true anyway if they consumer package had already updated to Hapi >= 19).